source/hassio/index.markdown

```diff
@@ -13,6 +13,10 @@ Hass.io turns your Raspberry Pi (or another device) into the ultimate home autom
 
 [Go to the installation instructions &raquo;][install]
 
+ <p class='note'>
+Home Assistant updates take time to roll into the Hass.io builds.  Therefore there is often a slight delay between the availability of a Home Assistant update and an update being available in Hass.io, be patient. See: [Upgrading](#upgrading)
+ </p>
+
```

Comment:
Either this note or the "Updating" section. I would prefer the "Updating" section. Both contains the same information.

Reply:
Yep, removed this change and kept the other.